utils.go

```diff
@@ -27,3 +29,13 @@ func waitForAliveVMM(ctx context.Context, client *Client) error {
 		}
 	}
 }
+
+// envValueOrDefaultTime check if env value exists and returns it or returns default value
+// provided as a second param to this function
+func envValueOrDefaultTime(envName string, def int) time.Duration {
```

Comment:
I think I'd prefer to see this just return an `int` (and named `envValueOrDefaultInt`) and let the conversion to a `time.Duration` occur elsewhere where it can be scaled (by `time.Millisecond` or `time.Second` as is happening for the two new environment variables).  Letting this return a `time.Duration` doesn't really make it clear as to the scale of the duration.